comms/fundraising/day-of-giving/README.md

```diff
@@ -0,0 +1,32 @@
+# Day of Giving
+
+The Purdue [Day of Giving](https://dayofgiving.purdue.edu/) is an annual
+fundraising event run by the Purdue for Life Foundation. It runs over the
+course of 24 hours and allows various alumni, students, and others invested
+in Purdue to donate to various causes and organizations at the university.
+
+We can sign up to participate in the Day of Giving by registering our
+club through an email sent via Boilerlink to the club's president and
+treasurer.
+
+Once we are registered, we get a link to our club's donation page, via which
+we can accept donations. There are also various [challenges](https://www.purdueforlife.org/alumni-leaders/purdue-day-of-giving-resources/)
+that occur during the Day of Giving, however it's damn near impossible to win
+them as a student organization (we can't compete with Purdue Bands for example).
+
+Money raised during the Day of Giving goes to the Purdue Hackers goes to the
+BOSO TooCOOL account after some indeterminate amount of time.
+
+## Strategy
+
+We have not yet determined a strategy for the Day of Giving, but we should
+probably do something to encourage donations from our alumni and friends.
+
+## History
```

Comment:
nit: since this ends with an important opinion it probably shouldn't be buried under "history". but it's not a big deal